honeybee_radiance/cli/dc.py

```diff
@@ -242,10 +242,10 @@ def rfluxmtx_command_with_postprocess(
 
         options = RfluxmtxOptions()
         # parse input radiance parameters
-        if rad_params.strip():
+        if rad_params:
```

Comment:
Checking with `.strip()` makes sure we handle cases like this one `     `. Is it a reason that you decided to remove them?

Reply:
If I don't use the option '--rad-params' in command line it fails. Without `.strip()` it works even with '--rad-params ""' and when I exclude the option completely.

![image](https://user-images.githubusercontent.com/34610298/159721041-50e74264-0568-4244-b283-de0065bfa1d0.png)

Reply:
Good catch!